src/python/pants/backend/jvm/targets/jvm_binary.py

```diff
@@ -85,13 +85,11 @@ def path(self):
 
     CONCAT_TEXT = "CONCAT_TEXT"
     """Concatenates the contents of all duplicate entries encountered in the order encountered,
-  separating entries with newlines if needed.
-  """
+    separating entries with newlines if needed."""
```

Comment:
It seems to me that the closing triple quote is consistently on a newline everywhere but here and at line 92. Was this on purpose?

Reply:
This was done by docformatter. Docformatter always has it on the same line when there is no description below.